src/backend/services/wallet-api/src/routes.py

```diff
@@ -27,7 +28,15 @@ def createWallet():
     if (data is None) or ("walletId" not in data):
         raise IncorrectPayloadException()
     
-    response = requests.post( blockchain_wallet_url.format("addWallet"), json=data)
+    walletId = data["walletId"]
+    response = send_get_request( user_api_url.format("umnetID/"+ walletId), None)
+    try:
+        user_data = response.json()
+        success = user_data["success"]
+    except KeyError as e:
+        raise UserNotFoundException()
+    
+    response = send_post_request( blockchain_wallet_url.format("addWallet"), data)
```

Comment:
I dont think I 100% understand what was changed here, can you explain these block a bit for me please

Reply:
This code block needs to be removed, it was essentially checking if the user is present in the db. Now that we want the wallet to be created when a user is created this won't be necessary.

Reply:
Ah ok that make more sense, thanks